scio/README.md

```diff
@@ -0,0 +1,18 @@
+Scio interpreter for Apache Zeppelin
+====================================
+
+## Raison d'être:
```

Comment:
I'm happy to change, that said `Raison d'être` is commonly used in English. What do you think?